src/components/Field.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Field = ({type, id, onChange, value, children}) => {
```

Comment:
I think I would use `label` prop instead of `children`.
Usually you would use `children` for something substantial. Here you are using it only for the label.